### DISCUSSION TITLE
What about autocompletion menu in code-server?

### DISCUSSION THREAD

**Discussion by bgachenot** [coder/code-server#2727] [Q&A] [answered]
Hello,

I am using this awesome application for about 6 months and as far as I can remember, I always had a small issue, annoying but not a big problem with the autocompletion.
Let me show you two screens:

Code-server autocompletion list:
![image](https://user-images.githubusercontent.com/62659109/107887457-9417b280-6f06-11eb-801b-d09089e5e2b9.png)
Vscode autocompletion list:
![image](https://user-images.githubusercontent.com/62659109/107887490-d17c4000-6f06-11eb-87f8-fcc2ee89eca2.png)

As you can see, the list on code-server is only one element, we can browse the list with <arrow up/down> or using the mouse but it is quite annoying (at least for me) and not very practical when I'm looking for a specific snippet for example.

Browsers used for testing this "bug": Google Chrome, Microsoft Edge (which is based on Chromium), Firefox
code-server version: 1.9.0
Operating system where code-server is installed: Fedora 32
Tested under a nginx reverse-proxy and with direct access but it is displayed the same as on the first screen.

Am I the only one facing this issue or did I missed a parameter or something else?

**Comment by nooobcoder** [comment, marked as answer]
@bgachenot  You might not have installed required extensions for that language to get suggestions support! You need to install the extensions first. Which language are you using to write code here, C++, Java?
Also, might be a problem with the intellisense, did you try reinstalling the code-server?
First of all, confirm you have extension installed for that particular language.
Please post a reply to this, so that I can assist you better.

**Reply by nooobcoder** [reply]
@bgachenot your reply is still awaited?
Please keep an eye on the thread when you have opened a discussion.

**Reply by bgachenot** [reply]
Hello,
I'm sorry, I wrote my answer yesterday but I must have forgotten to send it.

I tried with no extensions at all.
I also tried with a Python file with only the Python and Jupyter extensions.
And I tried on a C file with only the C/C++ extension (installed from the .vsix file from the github project).

The display is the same with all of these tests (picture 2).

I reinstalled code-server when I upgraded from 3.8.1 to 3.9.0,
I'm going to try with a fresh install on a fedora 32 VM to see if I have the same issue.

Thanks for you reply!

**Reply by nooobcoder** [reply]
@bgachenot did it work out?

**Reply by bgachenot** [reply]
@nooobcoder 
I tried code-server with Fedora 32 and Ubuntu 20.10 (LXC containers) and it is working without any issue. (Installing code-server, adding Python plugin and creating a basic Python file)
So I go back to my server, I remove entirely code-server (uninstalling code-server with dnf first then removing all the remaining folders in .config, .local/share and /usr/lib/) and I reinstall from scratch using the script.
It is now working as expected. I think that it was caused by an old broken installation of code-server or extension that created some broken files, leading to this particular issue. (It is now a bit faster too)

Thanks for the help!

**Reply by nooobcoder** [reply]
@bgachenot  ya, maybe the broken/corrupt config files were causing the issues, or maybe due to the cached broken extensions.
